event-bus/src/lib.rs

```diff
@@ -162,15 +168,31 @@ impl<'d> EventConsumer<'d> {
     }
 }
 
-#[derive(Clone, Debug, clap::Parser)]
+#[derive(Clone, Debug, clap::Parser, Default)]
 #[command(rename_all_env = "SCREAMING_SNAKE_CASE")]
 pub struct EventBusConfig {
     /// Event bus to configure
-    #[arg(long = "event-bus", value_enum, default_value = "kafka")]
+    #[arg(env = "EVENT_BUS_TYPE", long = "event-bus", value_enum, default_value = "kafka")]
```

Comment:
Unless there are very good reasons, I would like to keep long name and env-vars named alike.